README.md

```diff
@@ -1,194 +1,239 @@
-to the task definition will require changing the terraform in
-`transcom/transcom-infrasec-gov-nonato/transcom-gov-dev/app-dev/loadtesting.tf`.  This file is updated manually to
-reflect the current state.
+Each `User` class needs a set of tasks to complete to be able to run a load test. All tasks are
+callables that can be manually set into the `tasks` attribute of the user, or they can be organized
+into instances of the locust class `TaskSet` and then associated with a user. This directory
+contains all the tasks used in our load tests.
 
-### `scripts`
+To read more about users and tasks and how they interact, refer
+to [Writing a locustfile](https://docs.locust.io/en/stable/writing-a-locustfile.html).
 
-`aws-session-port-forward.py` - This is the script used to access the deployed locust load testing container and forward
-to your local port 4000 accessible at [http://localhost:4000](http://localhost:4000).
+### `utils/`
 
-`codebuild` - This script is invoked when making a new build/deployment using the AWS CodeBuild service.  It builds a
-new docker image and publishes it to ECR so the service can pull down the new image.  It also controls updating the
-service if there is a new task definition from updating the Terraform code.
+This directory contains python code and utilities that help us run our load tests. For example, the
+code we use to manage authenticating to the Prime API. Mixin classes, helper functions, and
+constants are located here.
 
 ## Getting Started
 
-*Note: These instructions include the relevant commands for MacOS only. Please keep this in mind and be prepared to
-search for alternatives if you are running a different OS.*
+*Note: These instructions include the relevant commands for MacOS only. Please keep this in mind and
+be prepared to search for alternatives if you are running a different OS.*
 
 ### Base Installation
 
-We have two supported installation methods, `pyenv` and `nix`. Pick which you prefer and proceed to that section.
+We have two supported installation methods, `pyenv` and `nix`. Pick which you prefer and proceed to
+that section.
 
 #### Setup: Pyenv and Pipenv
 
-When setting up for the first time, before you run `direnv allow`, run
+1. When setting up for the first time, before you run `direnv allow`, run
 
-  ```shell
-  make install_tools
-  ```
+    ```shell
+    make install_tools
+    ```
 
-This will install `pyenv` and `pipenv` along with other tools like `pre-commit`. Now run
+    1. This will install `pyenv` and `pipenv` along with other tools like `pre-commit`.
 
-  ```shell
-  direnv allow
-  ```
+2. Restart your terminal.
+3. Now run
 
-This should install the dependencies via `pipenv` automatically.
+    ```shell
+    direnv allow
+    ```
 
-If the base dependencies change, you can always re-run this command.
+   1. This should install the dependencies via `pipenv` automatically.
+   2. If the base dependencies change, you can always re-run this command.
```

Comment:
I think this is true the first time but I was generally having to run `pipenv install` unless I deactivated or destroyed the virtual environment.

Reply:
Ah interesting, yeah it looks like direnv only runs the install if there isn't already a venv: 
https://github.com/direnv/direnv/blob/f68f741895754e9551059c8e0132a796ad1c3c3a/stdlib.sh#L882-L904

I've updated the README with new instructions for updating python dependencies. Thanks for pointing this out!